addons/api/addon/serializers/account.js

```diff
@@ -52,9 +52,8 @@ export default class AccountSerializer extends ApplicationSerializer {
       serialized.attributes = {};
       const { login_name } = snapshot.record;
       if (login_name) serialized.attributes.login_name = login_name;
-    }
-    // LDAP does not support updating the auth method ID or type.
-    if (!snapshot.record.isNew) {
+    } else {
+      // OIDC does not support updating the readonly auth method ID or type attributes.
```

Comment:
None blocking comment: Why we are mentioning here OIDC and deleting fields if this is `serializeLDAP` ?

Reply:
good catch, this should be LDAP